include/odp/api/spec/ipsec.h

```diff
@@ -192,6 +232,28 @@ typedef struct odp_ipsec_outbound_config_t {
 
 } odp_ipsec_outbound_config_t;
 
+/**
+ * IPSEC TEST capability
+ */
+typedef struct odp_ipsec_test_capability_t {
+	/** Parameters supported for sa_update */
+	struct {
+		/** Next sequence number value
+		 *
+		 * @see ODP_IPSEC_TEST_SA_UPDATE_SEQ_NUM
+		 */
+		odp_bool_t seq_num;
+
+		/** Highest authenticated sequence number
+		 *
+		 * @see ODP_IPSEC_TEST_SA_UPDATE_ANTIREPLAY_WINDOW_TOP
+		 */
+		odp_bool_t antireplay_window_top;
+
+	} sa_update_params;
```

Comment:
I think it would be better to name this sa_operations to match the name of the enum that got renamed odp_ipsec_test_sa_operation_t.

Reply:
Agreed. Renamed to sa_operations in v7.